doc/source/reference/random/c-api.rst

```diff
@@ -0,0 +1,177 @@
+Cython API for random
+---------------------
+
+.. currentmodule:: numpy.random
+
+Typed versions of many of the `Generator` and `BitGenerator` can be accessed
```

Comment:
This is confusing. What does it mean?

Reply:
missing fragment. It should read "Typed versions of many of the methods and functions in `Generator` and `BitGenerator` can be accessed ...
Fixing